### PR TITLE
Feat/doc updates

### PR DESCRIPTION
**Pull request by janwilli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Refer to application package and dependency trackers for additional dependency i
   - [VersionEye](https://www.versioneye.com/user/projects/58a669e7b4d2a20055fcb84c)
   - [Bithound](https://www.bithound.io/github/nci-ats/fs-middlelayer-api/feat%2Fswagger-ui/dependencies/npm)
 
-The Moxai package is a dependency for testing and was built specifically for this project. Moxai was published as an [independent package](https://www.npmjs.com/package/moxai) that can be used with any Express application. This application uses the moxai package as a placeholder mock API. The /mocks/basic.json file maintains the API endpoint schema. 
+The [Moxai package](https://www.npmjs.com/package/moxai) is a dependency for testing and was built specifically for this project. Moxai was published as an [independent package](https://www.npmjs.com/package/moxai) that can be used with any Express application. This application uses the moxai package as a placeholder mock API. The [/mocks/basic.json file](mocks/basic.json) maintains the API endpoint schema.
```
Comment (lpendem) [inline README.md]: need not to be linked to npm twice here, you may want to remove link on 'independent package'.

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Refer to application package and dependency trackers for additional dependency i
   - [VersionEye](https://www.versioneye.com/user/projects/58a669e7b4d2a20055fcb84c)
   - [Bithound](https://www.bithound.io/github/nci-ats/fs-middlelayer-api/feat%2Fswagger-ui/dependencies/npm)
 
-The Moxai package is a dependency for testing and was built specifically for this project. Moxai was published as an [independent package](https://www.npmjs.com/package/moxai) that can be used with any Express application. This application uses the moxai package as a placeholder mock API. The /mocks/basic.json file maintains the API endpoint schema. 
+The [Moxai package](https://www.npmjs.com/package/moxai) is a dependency for testing and was built specifically for this project. Moxai was published as an [independent package](https://www.npmjs.com/package/moxai) that can be used with any Express application. This application uses the moxai package as a placeholder mock API. The [/mocks/basic.json file](mocks/basic.json) maintains the API endpoint schema. 
 
 The controllers/index.js file has random control number generation logic that should be removed when the mock API is replaced with the Basic API.
```
Comment (lpendem) [inline README.md]: No more valid, this paragraph needs to be removed.

File: docs/permit-creation.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ These steps define the process for creating a new permit type using Example Perm
...
+    1. If there are any new form fields not supported by the current middle-layer  database, they can be added in the application table. To do this, create a new migration file (e.g., `06-alter-applications.js`) with the sequelize alter table script and save it under `dba/migrations/`. Also, update `src/models/applications.js` to include the new database fields. Please refer to the [Sequelize migrations documentation](http://docs.sequelizejs.com/en/latest/docs/migrations/) for information on altering an existing table.
     2. If there are routing changes, update `src/controllers/index.js`.
     3. If there are validation changes, update `src/controllers/validation.js`.
```
Comment (lpendem) [inline docs/permit-creation.md]: update to add `src/controllers/fileValidation.js` as well.

File: docs/permit-creation.md [hunk 4]
```diff
@@ -154,7 +154,7 @@ These steps define the process for creating a new permit type using Example Perm
...
 
 2. Extend the schema, if necessary.
-    1. If there are any new form fields not supported by the current middle-layer  database, they can be added in the application table. To do this, go to `dba/migrations/ 02-create-applications.js` and update the sequelize migration script as needed. Also, update `src/models/applications.js` to include the new database fields. This applies only if you are recreating the applications table. Please refer to the [Sequelize migrations documentation](http://docs.sequelizejs.com/en/latest/docs/migrations/) for information on altering an existing table.
+    1. If there are any new form fields not supported by the current middle-layer  database, they can be added in the application table. To do this, create a new migration file (e.g., `06-alter-applications.js`) with the sequelize alter table script and save it under `dba/migrations/`. Also, update `src/models/applications.js` to include the new database fields. Please refer to the [Sequelize migrations documentation](http://docs.sequelizejs.com/en/latest/docs/migrations/) for information on altering an existing table.
```
Comment (lpendem) [inline docs/permit-creation.md]: more than one space here in between 'middle-layer  database'

Comment (lpendem) [inline docs/permit-creation.md]: permit-creation.md file has two references to code files as 'src/controllers/..' that are not quoted.

**Comment by janwilli** [comment]
Fixes made

**Pull request merged**